spec/factories.rb

```diff
@@ -3,7 +3,7 @@
   factory :person do
     first_name 'John'
     last_name 'Doe'
-    email "john.doe#{Random.rand(1000)}@example.com"
+    sequence(:email) { |n| "john.doe#{n}@example.com" }
```

Comment:
This doesn't work in the test environment. If you attempt to use this style of get a Validation failed: Email has already been taken error. I've googled a bunch and tried to debug it and every source I find recommends doing things this way so I tried it and it solved the test environment issues I was having.